life_expectancy/cleaning.py

```diff
@@ -1,48 +1,72 @@
 """ This module cleans the data"""
-
 import pandas as pd
 import numpy as np
+from life_expectancy.countries import Region
 
-
-def clean_data(
-        life_exp_raw_data: pd.DataFrame,
-        region: str = 'PT'
-) -> pd.DataFrame:
-    """ 
-    Cleans the life expectancy data by split columns,
-    filter the region, unpivot the years to long format and
+def clean_data (life_exp_raw_data: pd.DataFrame):
+    """
+    Cleans the life expectancy data by unpivot the years to long format and
     ensure year is an integer and value is a float type
     Args:
         life_exp_raw_data(Pandas DataFrame): DataFrame with data to be cleaned
-        region (str): String with region information
-
     Returns: 
-        life_exp (Pandas DataFrame): DataFrame with life expectancy cleaned data
-    """
+        life_exp (Pandas DataFrame): DataFrame with life expectancy cleaned data"""
 
     # Split column with unit, sex, age, geo to multiple columns
+    column_split = 'unit,sex,age,geo\\time'
     columns = ["unit", "sex", "age", "region"]
-    life_exp_raw_data[columns] = (life_exp_raw_data.iloc[:,0].str.split(',', expand = True))
-    life_exp = life_exp_raw_data.drop(columns=life_exp_raw_data.columns[0])
 
-    # Filters only the data for the region selected
-    life_exp = life_exp.loc[life_exp["region"] == region]
+    life_exp_raw_data[columns] = life_exp_raw_data[column_split].str.split(',', expand = True)
+    life_exp = life_exp_raw_data.drop(columns=[column_split])
 
     # Unpivot the years to long format
     life_exp = life_exp.melt(
         id_vars = columns,
         var_name = "year",
         value_name = "value"
         )
-
     # Ensure year is an int
     life_exp["year"] = life_exp["year"].str.replace(" ", "").astype(int)
 
     # Remove letters, whitespaces and replace ":" by NaN
     life_exp["value"] = life_exp["value"].str.replace(r"[a-zA-Z\s]+", "",  regex=True)
     life_exp["value"] = life_exp["value"].replace(":", np.NaN)
+
     # Ensure value is a float and drop NA values
     life_exp["value"] = life_exp["value"].astype(float)
+
     life_exp = life_exp.dropna(subset=["value"])
 
     return life_exp
+
+def feature_cleaning(life_exp: pd.DataFrame):
```

Comment:
This name is a bit generic and it's ambiguous with the `clean_data` function. Why not call it `rename_and_drop_cols` instead. Remember a long and descriptive name is better than a short and cryptic one

Reply:
Thanks! Done :)